sgl-kernel/csrc/moe/fused_qknorm_rope_kernel.cu

```diff
@@ -195,43 +195,49 @@ __global__ void fusedQKNormRopeKernel(
   float cos_vals[numElemsPerThread];
   float sin_vals[numElemsPerThread];
   float pos_id = static_cast<float>(position_ids[tokenIdx]);
+  int const rotary_lanes = rotary_dim / numElemsPerThread;  // rotary range
+  bool const applyRotary = (laneId < rotary_lanes);
+  if (applyRotary) {
```

Comment:
Can we use guard clause here to avoid nested if else?
```
if (!applyRotary) return;

// main logic goes here.
```

Reply:
The unrotated part still needs to be written back to qkv buffer, so we can not return. 

If it is not clear enough, maybe we can encapsulate the rotation part in an inline function?